java/ql/lib/semmle/code/java/frameworks/spring/SpringController.qll

```diff
@@ -156,6 +156,11 @@ class SpringRequestMappingMethod extends SpringControllerMethod {
   /** Gets the "value" @RequestMapping annotation value, if present. */
   string getValue() { result = requestMappingAnnotation.getStringValue("value") }
 
+  /** Gets the "method" @RequestMapping annotation value, if present. */
+  string getMethod() {
```

Comment:
This isn't getting a `Method`, so it's a bit confusing. How about renaming like this:
```suggestion
  string getMethodValue() {
```

Reply:
Update in [0d41a1c](https://github.com/github/codeql/pull/18288/commits/0d41a1c15789521b11d26b055fa385471a0d69ae)